fili-core/src/main/java/com/yahoo/bard/webservice/table/availability/PermissiveAvailability.java

```diff
@@ -26,36 +25,37 @@
     /**
      * Constructor.
      *
-     * @param tableName The name of the data source
-     * @param columns A set of columns associated with the data source
-     * @param metadataService A service containing the data source segment data
+     * @param tableName  The name of the data source
+     * @param metadataService  A service containing the data source segment data
      */
     public PermissiveAvailability(
             @NotNull TableName tableName,
-            @NotNull Set<Column> columns,
             @NotNull DataSourceMetadataService metadataService
     ) {
-        super(tableName, columns, metadataService);
+        super(tableName, metadataService);
     }
 
     /**
      * Returns union of all available intervals.
      * <p>
      * This is different from its parent's
      * {@link
-     * com.yahoo.bard.webservice.table.availability.ConcreteAvailability#getAvailableIntervals(DataSourceConstraint)};
+     * com.yahoo.bard.webservice.table.availability.ConcreteAvailability#getAvailableIntervals(
+     * PhysicalDataSourceConstraint
+     * )};
      * Instead of returning the intersection of all available intervals, this method returns the union of them.
      *
-     * @param ignoredConstraints  Data constraint containing columns and api filters. Constrains are ignored, because
+     * @param constraint  Data constraint containing columns and api filters. Constrains are ignored, because
```

Comment:
Why was this changed?  I'm pretty sure that the original 'ignored' was intentional.  
Also, spelling 'Constraints'